tests/unit/json_test.c

```diff
@@ -922,6 +922,7 @@ static void test_parse_escaped_string(void)
 static void test_parse_big_numbers(void)
 {
 #define JSON_TEST_BIG_NUMBER "9999999999"
+#define JSON_TEST_BIG_NUMBER_INT64 9999999999
```

Comment:
Should be `9999999999L`, I think.

Reply:
Fixed with the `LL`  suffix, as discussed in private.